backend/plan/urls.py

```diff
@@ -15,6 +16,7 @@
 router.register(r"primary-schedules", PrimaryScheduleViewSet, basename="primary-schedules")
 
 urlpatterns = [
+    path("scheduler/courses/", AutomaticCourseScheduler.as_view(), name="automatic-scheduler"),
```

Comment:
Probably don't need the `/courses/` unless you think there is some collision that might come up.

Reply:
Fixed.